backend/src/routes/api/validate-isv/validateISV.ts

```diff
@@ -24,8 +23,11 @@ const waitOnDeletion = async (reader: () => Promise<void>) => {
   }
 };
 
-const waitOnCompletion = async (reader: () => Promise<boolean>): Promise<boolean> => {
-  const MAX_TRIES = 60;
+const waitOnCompletion = async (
+  fastify: KubeFastifyInstance,
+  reader: () => Promise<{ valid: boolean; error: string }>,
+): Promise<{ valid: boolean; error: string }> => {
+  const MAX_TRIES = 120;
```

Comment:
parallel alternative: might be a candidate for a dotenv value/setting

Reply:
Noted. We can add that later if we think its necessary.